From a31c68322d1e59dcfc34b5f8ae0f701d43e539c5 Mon Sep 17 00:00:00 2001
From: andrei varvara <andrei.varvara@freescale.com>
Date: Mon, 21 May 2012 22:29:27 +0000
Subject: [PATCH 199/518] Minor cosmetics fix. Tab added in disable_sa
 function

cosmetics. add a tab.

Signed-off-by: Andrei Varvara <andrei.varvara@freescale.com>
Acked-by: Mihai Serb <mihai.serb@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dpa_ipsec.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
index 6a87000..fefb613 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
@@ -4018,8 +4018,8 @@ int dpa_ipsec_disable_sa(int sa_id)
 	sa = get_sa_from_sa_id(sa_id);
 	if (!sa) {
 		xx_pr_err("Invalid SA handle for SA id %d\n", sa_id);
-	return -EINVAL;
-}
+		return -EINVAL;
+	}
 
 	/* Acquire protective lock for this SA */
 	ret = mutex_trylock(&sa->lock);
-- 
1.7.5.4

